stylesheets/stylesheet.css

```diff
@@ -477,6 +477,33 @@ Custom Styles
   margin-bottom: 0;
 }
 
+.projects .label .count {
+  background: none white;
+  border: 1px solid white;
+  border-radius: 8px;
+  color: #0090ff;
+  display: inline-block;
+  height: 16px;
+  line-height: 16px;
+  margin: -2px -4px 0 4px;
```

Comment:
Hrm, this is what I see in Chrome (Windows):
![2015-02-09_03-49-44](https://cloud.githubusercontent.com/assets/133987/6103964/c35b43a8-b00e-11e4-8e6f-308a99d106bb.png)

And Firefox:
![2015-02-09_03-47-36](https://cloud.githubusercontent.com/assets/133987/6103926/777efdb2-b00e-11e4-81c5-066457251558.png)
And IE:
![2015-02-09_03-48-42](https://cloud.githubusercontent.com/assets/133987/6103945/93db93bc-b00e-11e4-8da6-178c1e93ede4.png)

All that said, I'm no CSS Guru either - styling was set for the spinner image first, then adjusted for the count. Probably better to go the other direction.